packages/tools/src/tools/base/ContourSegmentationBaseTool.ts

```diff
@@ -207,7 +212,12 @@ abstract class ContourSegmentationBaseTool extends ContourBaseTool {
     let lineOpacity = 1;
     let fillOpacity = 0;
 
-    if (active) {
+    if (autoGenerated) {
```

Comment:
I dislike getting these in this way - would prefer to have the style getter algorithms directly retrieve the data, but I don't want to make that change in this PR.